README.md

```diff
@@ -109,22 +109,21 @@ $ pytest
 4. Read through the test failure, and understand why the failure is happening. Confirm your findings with a classmate. 
     - If it is a test you wrote, consider whether you are actually testing what you intend to test. Does the test need modification?
 
-
 5. Make a plan to implement code to pass the test.
 
 6. Write code in `party.py` to pass the test.
 
 7. Re-run the tests.
 
-8. Repeat steps 3-7 until that test passes! (GREEN)
+8. Repeat steps 3-7 until that test passes!
 
-9. Repeats steps 2-8 until you have finished all tests in the file.
+9. Repeat steps 2-9 until you have finished all tests in the file.
```

Comment:
Nice catch on the Repeat case, I think here the numbers should still be 2-8 since we haven't added a new step as with swap-meet.

Reply:
Ah! There was a difference in where "Consider looking for opportunities to improve your code" was. I moved it to step 10 for viewing party and swap meet